pages/index.js

```diff
@@ -78,6 +78,19 @@ app.get("/upload-document", function (req, res) {
   })
 });
 
+app.get("/upload-document-for-existing-customer", function (req, res) {
```

Comment:
@ShreyaThapa where is this route being used?

Reply:
It isn't being used explicitly, but provides an example for users if they want to use the drop-ins for existing customers. I missed out on adding a comment to explain that. 

Let me know if you think this might actually more confusing than helpful!

Reply:
yeah, that may be a good idea. Let's add a comment above that describes what this is doing and how it can be used. Also, should we change line 86 where it references replacing an id in production? Shouldn't they replace that ID regardless of whether they are copy+ pasting that code and using in Sandbox or Prod?

Reply:
That makes sense. Added a comment and updates existing comments where needed in [this commit](https://github.com/Dwolla/drop-ins-examples/pull/15/commits/bbbb5eb16244f1dc8977c5bc8f648c00ea0b698f)!